File: crates/axhub-helpers/tests/fixtures/site-scan/java/bad.java
package sample;

import java.net.http.HttpClient;
import java.sql.DriverManager;

class Sample {
    void load() throws Exception {
        var http = HttpClient.newHttpClient();
        var conn = DriverManager.getConnection("jdbc:postgresql://db/x");
        String url = "https://api.axhub.dev/v1/posts";
        System.out.println(http + conn + url);
    }
}

File: crates/axhub-helpers/tests/fixtures/site-scan/kotlin/bad.kt
package sample

import java.net.http.HttpClient
import java.sql.DriverManager

fun load() {
    val http = HttpClient.newHttpClient()
    val conn = DriverManager.getConnection("jdbc:postgresql://db/x")
    val url = "https://api.axhub.dev/v1/posts"
    println("$http $conn $url")
}
